repository/src/main/java/com/epam/esm/constructor/QueryConstructor.java

```diff
@@ -0,0 +1,100 @@
+package com.epam.esm.constructor;
+
+import com.epam.esm.constant.SqlGiftCertificateQuery;
+import org.springframework.stereotype.Component;
+
+@Component
+public class QueryConstructor {
+
+    private static final String SELECT_CERTIFICATES_WHERE_MATCH_TAG_NAME_FiRST_PART = "t.name IN ('";
```

Comment:
Could you please rename it with java naming conventions? https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
rename  Constant